src/components/KeyblockDetailsPanel.vue

```diff
@@ -0,0 +1,220 @@
+<template>
+  <app-panel class="keyblock-details-panel">
+    <header class="keyblock-details-panel__header">
+      <h2 class="keyblock-details-panel__heading h3">
+        DETAILS
+      </h2>
+      <div>
+        <copy-chip
+          :label="keyblockDetails.hash"
+          class="keyblock-details-panel__chip"/>
+        <copy-chip
+          :label="formatEllipseHash(keyblockDetails.hash)"
+          :clipboard-text="keyblockDetails.hash"
+          class="keyblock-details-panel__chip-ellipse"/>
+      </div>
+    </header>
+    <table>
+      <tbody>
+        <tr class="keyblock-details-panel__row">
+          <th class="keyblock-details-panel__table-header">
+            Keyblock height
```

Comment:
As for the unification of hints (and labels), we agreed that all labels should be capitalized. The content of hints should not be capitalized
```suggestion
            Keyblock Height
```